src/uct/ib/rc/base/rc_iface.h

```diff
@@ -485,8 +485,6 @@ uct_rc_iface_add_cq_credits_dispatch(uct_rc_iface_t *iface, uint16_t cq_credits)
     ucs_assertv(iface->tx.cq_available <= iface->config.tx_cq_len,
                 "cq_available=%d tx_cq_len=%d cq_credits=%d",
                 iface->tx.cq_available, iface->config.tx_cq_len, cq_credits);
-    ucs_arbiter_dispatch(&iface->tx.arbiter, 1, uct_rc_ep_process_pending,
-                         NULL);
 }
```

Comment:
now the name of this func is misleading, because no dispatch is done here

Reply:
fixed